sqlmesh/core/macros.py

```diff
@@ -648,6 +648,48 @@ def eval_(evaluator: MacroEvaluator, condition: exp.Condition) -> t.Any:
     return evaluator.eval_expression(condition)
 
 
+@macro()
+def star(
+    evaluator: MacroEvaluator,
+    relation: exp.Table,
+    alias: t.Optional[exp.Identifier] = None,
+    except_: t.Optional[exp.Array] = None,
+    prefix: t.Optional[exp.Literal] = None,
```

Comment:
you could make this Literal.string("") then down where you use it, you can just do prefix.name